src/utils/dataFormatter.ts

```diff
@@ -7,22 +7,24 @@ import { TestActivity } from './testActivity';
 
 export const formatDynamoData = (record: DynamoDBRecord): TestActivity[] => {
   const data = DynamoDB.Converter.unmarshall(record.dynamodb.NewImage);
-  const testActivities: TestActivity[] = data.testTypes.map((testType) => ({
-    noOfAxles: data.noOfAxles as number,
-    testTypeStartTimestamp: data.testStartTimestamp,
-    testTypeEndTimestamp: data.testEndTimestamp,
-    testStationType: data.testStationType,
-    testCode: testType.testCode,
-    vin: data.vin,
-    vrm: data.vrm,
-    testStationPNumber: data.testStationPNumber,
-    testResult: testType.testResult,
-    certificateNumber: testType.certificateNumber,
-    testTypeName: testType.name,
-    vehicleType: data.vehicleType,
-    testerName: data.testerName,
-    testerStaffId: data.testerStaffId,
-    testResultId: data.testResultId,
-  }));
+  const testActivities: TestActivity[] = data.testTypes
+    .filter(() => data.testStatus !== 'cancelled')
```

Comment:
Should we be using enums for values of the testStatus?

Also this seems like logic that's not exactly around formating data, what would happen if you wanted to reuse this formatter in future to get formatted cancelled data?

Reply:
The testStatus is a string and comes in from the DynamoDB stream. We don't have control over that.

I believe the formater will only have one use case. We could easily refactor later if we did want to start getting different events based on testStatus or some other property.

Reply:
The more I thought about it, the more it bothered me :-)
Renamed a few things to better reflect their usage.